lab_6_pipeline/pipeline.py

```diff
@@ -189,6 +299,13 @@ def _count_frequencies(self, article: Article) -> dict[str, int]:
         Returns:
             dict[str, int]: POS frequencies
         """
+        sentences_features = {}
+        for conllu_sentence in self._analyzer.from_conllu(article).sentences:
+            for word in conllu_sentence.words:
+                word_feature = word.to_dict()['upos']
+                sentences_features[word_feature] = str(self._analyzer.from_conllu(
```

Comment:
you read conllu for each word in each sentence of conllu document. do you feel something strange?

Reply:
indeed, fixed it